### DISCUSSION TITLE
Pseudotime for multiple cell fate decisions

### DISCUSSION THREAD

**Discussion by yiyelinfeng** [GreenleafLab/ArchR#705] [Questions / Documentation]
Hello ArchR group,

        Thanks for nice tools to analysze scATAC-seq data, Pseudotime inference when use ArchR with function "addTrajectory", there the parameter “trajectory” need input the order of cell groups to be used for constraining the initial supervised fitting procedure. how about multiple cell fate? such as HSC to MPP, MPP to LMPP, LMPP to CLP, LMPP to GMP, MPP to MEP, the CLP, GMP, MEP belong to different branches, how to input "trajectory" to plot the trajectory in a UMAP graph. 

         thanks!

**Comment by rcorces** [comment]
Thanks for using ArchR!
1. This is not possible using the standard ArchR trajectory methods but we have implemented ArchR to use monocle3 as well which I believe should be able to handle this. see `getMonocleTrajectories()` https://www.archrproject.com/reference/getMonocleTrajectories.html
2. Please note that submitting issues is reserved for bug reports. Questions on usage should be posted in the Discussions forum. I am migrating this issue there.
